conans/test/unittests/client/conf/config_installer/test_install_folder.py

```diff
@@ -0,0 +1,22 @@
+# coding=utf-8
+
+import unittest
+import os
+from os import listdir
+from os.path import isfile, join
+
+from conans.client.conf.config_installer import tmp_config_install_folder
+from conans.test.utils.tools import TestClient
+
+
+class InstallFolderTests(unittest.TestCase):
+
+    def test_unique_install_folder(self):
+        client = TestClient()
+
+        with tmp_config_install_folder(client.cache) as tmp_folder_first:
+            open(os.path.join(tmp_folder_first, "foobar.txt"), "w+")
+            with tmp_config_install_folder(client.cache) as tmp_folder_second:
+                first = [f for f in listdir(tmp_folder_first) if isfile(join(tmp_folder_first, f))]
+                second = [f for f in listdir(tmp_folder_second) if isfile(join(tmp_folder_second, f))]
+                self.assertEqual(first, second)
```

Comment:
At the end, after finishing the second context, a `rmdir(tmp_folder)` is called, and it should raise a `OSError` (directory doesn't exist), isn't it?

Reply:
no because our `rmdir` ignores when a file is not found.

Reply:
oh! sorry, I thought it was the `os` provided one.